lib/active_model/serializer/reflection.rb

```diff
@@ -42,17 +42,17 @@ def initialize(*)
 
       def link(name, value = nil, &block)
         @_links[name] = block || value
-        nil
+        :nil
```

Comment:
Not sure if this should be called `:nil` or something else. I actually wanted to have your input on this as I was not too creative here.